runtime/tesla/org.eclipse.rcptt.tesla.swt/src/org/eclipse/rcptt/tesla/internal/ui/player/TeslaSWTAccess.java

```diff
@@ -526,9 +528,9 @@ public static Menu getCTabFolderItemListMenu(CTabFolder tabFolder) {
 
 	public static int getRunnables(Synchronizer synchronizer) {
 		try {
-			Field locationField = Synchronizer.class.getDeclaredField("messageCount");
```

Comment:
Do we look compatibility with older versions of Eclipse?

Reply:
FrameworkUtil.getBundle(CTabFolder.class).getBundleId() should allow you @pzi-dsa to check the version of the class and only change the field if the version is after the changed field name.